### ISSUE TITLE
[4.0] Atum a11y colour contrast

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#14763]
As shown in the screenshots below the colour of the icon for the page title has been changed from white in J3 to #285783 in Atum
<img width="192" alt="screenshotr11-56-06" src="https://cloud.githubusercontent.com/assets/1296369/24080609/0fa13476-0c9b-11e7-8ac9-146069c8c12b.png">

<img width="211" alt="screenshotr11-55-59" src="https://cloud.githubusercontent.com/assets/1296369/24080608/0f7c0c1e-0c9b-11e7-91c8-274062a8f803.png">

This new colour contrast fails the a11y contrast checkers

Even without the contrast checker I found it very hard to see the icon

**Comment by ylahav** [comment]
From the point of a11y because we have an alternative (the text) ... BUT I must agree that the image is not clear enough...

**Comment by brianteeman** [comment]
a11y is not just about making accommodations - we should be aiming for a11y by default 

Note the icon itself should have aria-hidden=true or a screenreader will read out the unicode value of the icon but i am dealing with that in the J3 branch

**Comment by ylahav** [comment]
Any way, we are working all accessibility and we will fixed it

**Comment by ciar4n** [comment]
PR created... https://github.com/joomla/joomla-cms/pull/14770

**Comment by brianteeman** [comment]
Closed as there is a pr
